torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -1033,6 +1033,50 @@ def _get_state_dict(self, dl_kwargs):
 """  # noqa: E501
 
 
+VOXPOPULI_ASR_BASE_10K_EN = Wav2Vec2ASRBundle(
+    'wav2vec2_voxpopuli_base_10k_asr_en.pt',
+    {
+        "extractor_mode": "group_norm",
+        "extractor_conv_layer_config": [
+            (512, 10, 5),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 3, 2),
+            (512, 2, 2),
+            (512, 2, 2),
+        ],
+        "extractor_conv_bias": False,
+        "encoder_embed_dim": 768,
+        "encoder_projection_dropout": 0.0,
+        "encoder_pos_conv_kernel": 128,
+        "encoder_pos_conv_groups": 16,
+        "encoder_num_layers": 12,
+        "encoder_num_heads": 12,
+        "encoder_attention_dropout": 0.0,
+        "encoder_ff_interm_features": 3072,
+        "encoder_ff_interm_dropout": 0.1,
+        "encoder_dropout": 0.0,
+        "encoder_layer_norm_first": False,
+        "encoder_layer_drop": 0.1,
+        "aux_num_out": 28
+    },
+    _labels=utils._get_vp_en_labels(),
+    _sample_rate=16000,
+    _remove_aux_axis=(1, 2, 3, 31),
+)
+VOXPOPULI_ASR_BASE_10K_EN.__doc__ = """wav2vec 2.0 model with "Base" configuration.
```

Comment:
@mthrok fyi this doc format is inconsistent from the other ones (missing the extra line breaks between sentences, see [here](https://pytorch.org/audio/main/pipelines.html#torchaudio.pipelines.VOXPOPULI_ASR_BASE_10K_EN))